scipy/stats/_qmc.py

```diff
@@ -705,23 +720,53 @@ def __init__(
         self.rng_seed = copy.deepcopy(seed)
         self.num_generated = 0
 
+        config = {
+            # random-cd
+            "n_nochange": 100,
+            "n_iters": 10_000,
+            "rng": self.rng,
+
+            # lloyd
+            "tol":  1e-5,
+            "maxiter":  10,
+            "qhull_options": None,
+        }
+        self.optimization_method = _select_optimizer(optimization, config)
+
     @abstractmethod
-    def random(self, n: IntNumber = 1) -> np.ndarray:
+    def _random(
+        self, n: IntNumber = 1, *, workers: IntNumber = 1
+    ) -> np.ndarray:
+        ...
+
+    def random(
+        self, n: IntNumber = 1, *, workers: IntNumber = 1
+    ) -> np.ndarray:
         """Draw `n` in the half-open interval ``[0, 1)``.
 
         Parameters
         ----------
         n : int, optional
             Number of samples to generate in the parameter space.
             Default is 1.
+        workers : int, optional
```

Comment:
This param was (and still is) only available for `Halton` before and now because the `random` is shared, it has to be documented there. From a design perspective it's also better to have the same interface no matter the child class (IDE was complaining about this actually).